src/amo/components/UserProfile/styles.scss

```diff
@@ -20,10 +20,15 @@ $avatar-size: 64px;
 }
 
 .UserProfile-avatar {
-  display: block;
   height: $avatar-size;
   margin: 0 0 12px;
+  overflow: hidden;
   width: $avatar-size;
+
+  .UserAvatar-image {
```

Comment:
This doesn't need to be nested anymore, and we tend to avoid nested selectors to keep things simple 😄